src/lib/audit-logger.js

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+const fetch = require('node-fetch')
+const fs = require('fs')
+const path = require('path')
+const chalk = require('chalk')
+
+const OPERATIONS = {
+  AB_APP_DEPLOY: 'ab_app_deploy',
+  AB_APP_UNDEPLOY: 'ab_app_undeploy',
+  AB_APP_TEST: 'ab_app_test', // todo : remove after testing
+  AB_APP_ASSETS_DEPLOYED: 'ab_app_assets_deployed',
+  AB_APP_ASSETS_UNDEPLOYED: 'ab_app_assets_undeployed'
+}
+
+const AUDIT_SERVICE_ENPOINTS = {
+  stage: 'https://adp-auditlog-service-stage.adobeioruntime.net/api/v1/web/audit-log-api/event-post',
+  prod: 'https://adp-auditlog-service-prod.adobeioruntime.net/api/v1/web/audit-log-api/event-post'
+}
+
+/**
+ * Send audit log events to audit service
+ * @param {string} accessToken valid access token
+ * @param {object} logEvent logEvent details
+ * @param {string} env valid env stage|prod
+ */
+async function sendAuditLogs (accessToken, logEvent, env = 'prod') {
+  const url = AUDIT_SERVICE_ENPOINTS[env]
```

Comment:
This could be null, what do you do if your not passed 'prod' or 'stage'
You should just default to prod ...
```
const url = env === 'stage' ? AUDIT_SERVICE_ENPOINTS.stage : AUDIT_SERVICE_ENPOINTS.prod
```